### PR TITLE
Remove unused vim plugins

### PR DESCRIPTION
**Pull request by iures**
Removed Plugins:
```
'altercation/vim-colors-solarized'
'morhetz/gruvbox'
'fholgado/minibufexpl.vim'
'mileszs/ack.vim'
'vim-scripts/vim-terraform'
'rodjek/vim-puppet'
```

### CODE REVIEW

File: vimrc [hunk 1]
```diff
@@ -50,11 +45,8 @@ Bundle 'tpope/vim-rails'
 Bundle 'burnettk/vim-angular'
 Bundle 'othree/html5.vim'
 Bundle 'quentindecock/vim-cucumber-align-pipes'
-Bundle 'vim-scripts/vim-terraform'
-Bundle 'rodjek/vim-puppet'
 
 " misc
-Bundle 'tpope/vim-unimpaired'
```
Comment (benseligman) [inline vimrc]: i use this one a bunch, but i'm happy to put it in my custom if no one else does

Comment (benseligman) [inline vimrc]: `[<space>` and `]<space>` for adding lines above/below current without going into insert mode and `[n` and `]n` for finding conflict markers are the things i use most

**Comment by benseligman** [comment]
lgtm

File: vimrc (hunk 1 above)
Comment (iures) [inline vimrc]: Yeah, this is definitely one I'll keep myself.

**Comment by michaeldauria** [comment]
:+1:

**Pull request closed without merging**